docs/reference/commandline/service_logs.md

```diff
@@ -32,6 +32,76 @@ Options:
 
 The `docker service logs` command batch-retrieves logs present at the time of execution.
 
+## Examples
```

Comment:
Can you update the command description (https://github.com/docker/docker/blob/c7995fdc7760c23cdbed0fa4ca229517c71dba82/cli/command/service/logs.go#L35-L36)

and update the output here accordingly? Something like

```
Usage:  docker service logs [OPTIONS] SERVICE|TASK

Fetch the logs of a service or task
```